distribution/std-lib/Standard/src/Visualization/Helpers.enso

```diff
@@ -1,8 +1,37 @@
 from Standard.Base import all
 
+import Standard.Table.Data.Column
+import Standard.Table.Data.Storage
+import Standard.Table.Data.Table
+
+## PRIVATE
+Any.catch_ ~val = this.catch (_-> val)
```

Comment:
This definition is silly. You can only dispatch a method on an error if it's defined on it directly, so this will never execute code and is just equivalent to `Any.catch_ _ = this`. Furthermore, please add type signatures for all functions in this file.

Reply:
> This definition is silly. 

It's not. :P

The responsibility of `catch_` is to invoke `catch` and ignore the error in the handler.
The fact that code is not executed is actually part of logic of `catch`, not `catch_`. If `catch` decided to do something on non-error values (like log for some debug purposes or handling some special, internal Any-like errors), the `catch_` should do the same.
Thus I prefer delegation rather than delegating in one case and reimplementing in the other.

Reply:
Fair enough!